spec/monitoring/middleware/conjur_collector_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+require 'rack/test'
+require 'monitoring/prometheus'
+require 'monitoring/middleware/conjur_collector'
+require 'monitoring/middleware/prometheus_exporter'
+
+describe Monitoring::Middleware::ConjurCollector do
```

Comment:
If you haven't come across it yet, [BetterSpec](https://www.betterspecs.org/) is a great resource for RSpec best practices.